tests/test_main.py

```diff
@@ -59,6 +59,20 @@ class Model(BaseModel):
     m = Model(a=10.2)
     assert m.a == 10.2
     assert m.b == 10
+    assert m.__doc__ is None
+
+
+def test_success_docstring():
+    class Model(BaseModel):
+        """Another model"""
+
+        a: float
+        b: int = 10
+
+    m = Model(a=10.2)
+    assert m.a == 10.2
+    assert m.b == 10
+    assert m.__doc__ == 'Another model'
```

Comment:
We can leave out this addition to the tests, as nothing here is affected by the changes to `create_model`

Reply:
Thank you! I have addressed this [here](https://github.com/pydantic/pydantic/pull/7863/commits/291dd3ad68f3fb1b6023acc6bd36d868bd717824).